pwndbg/heap/ptmalloc.py

```diff
@@ -288,11 +391,46 @@ def tcache_next_offset(self):
     def get_heap(self, addr):
         raise NotImplementedError()
 
+    def get_first_chunk_in_heap(self, heap_start=None):
+        # Find which arena this heap belongs to
+        if heap_start:
+            arena = self.get_arena_for_chunk(heap_start)
+        else:
+            arena = self.get_arena()
+
+        ptr_size = self.size_sz
+
+        start_addr = heap_start
+        if arena != self.main_arena:
+            start_addr += self.heap_info.sizeof
+            if pwndbg.vmmap.find(self.get_heap(heap_start)['ar_ptr']) == heap_region:
```

Comment:
I think maybe we should add something like: `heap_region = self.get_heap_boundaries(heap_start)` before this line to avoid `NameError`?

Reply:
What NameError can we get here?

Reply:
Ah, sorry I didn't express myself clearly.
I mean, name `heap_region` is not defined here?